dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +73,25 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @RolesAllowed({ "admin", "curator"})
+    @Path("/user/{userid}")
+    @Operation(operationId = "getUserLambdaEvents", description = "Get all of the Lambda Events for the given user and GitHub organization.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    public List<LambdaEvent> getUserLambdaEvents(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
+           @PathParam("userid") long userid,
+           @QueryParam("offset") @DefaultValue("0") String offset,
+           @DefaultValue("1000") @QueryParam("limit") Integer limit) {
+        final User user = userDAO.findById(userid);
```

Comment:
It's possible that the provided userid is invalid, so the user doesn't exist. Should check if null and throw error